src/System Application/App/Permission Sets/src/PermissionImpl.Codeunit.al

```diff
@@ -97,30 +99,40 @@ codeunit 9864 "Permission Impl."
                             if TenantPermission."Read Permission" <> PermissionOption then begin
                                 TenantPermission."Read Permission" := PermissionOption;
                                 ModifyPermissionLine := true;
+                                Session.LogAuditMessage(StrSubstNo(PermissionUpdatedLbl, RIMDX, TenantPermission."App ID", TenantPermission."Role ID", TenantPermission."Object Type", TenantPermission."Object ID",
+                                    TenantPermission."Read Permission", UserSecurityId()), SecurityOperationResult::Success, AuditCategory::RoleManagement, 2, 0);
```

Comment:
0: AdministeredEnvironment or 3: AdministeredCompany

Reply:
Defined by PMs, so will keep it and sync up with them.